sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/plans/logical/statsEstimation/SizeInBytesOnlyStatsPlanVisitor.scala

```diff
@@ -67,7 +67,7 @@ object SizeInBytesOnlyStatsPlanVisitor extends LogicalPlanVisitor[Statistics] {
     }
   }
 
-  override def visitDistinct(p: Distinct): Statistics = default(p)
+  override def visitDistinct(p: Distinct): Statistics = visitUnaryNode(p)
```

Comment:
Do we have a test coverage for this change? The newly added test case seems to pass without this change.

Reply:
Just to be consistent with the logic of `BasicStatsPlanVisitor`. The result will not change:
![image](https://user-images.githubusercontent.com/5399861/122626418-12ff4080-d0dd-11eb-841a-49c356b8338e.png)